bin/hledger-prices.hs

```diff
@@ -19,9 +20,10 @@ import Hledger.Cli
 ------------------------------------------------------------------------------
 cmdmode = hledgerCommandMode
   [here| prices
-Print all prices from the journal.
+Print all market prices from the journal.
   |]
-  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings instead of market prices"]
+  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings"
+  ,flagNone ["inverted-costs"] (setboolopt "inverted-costs") "print transaction inverted prices from postings also"]
   [generalflagsgroup1]
```

Comment:
I found it not obvious what "inverted" or "cost" meant here, and had to study the tests. Using "cost" is understandable since it appears in the -B/--cost-basis flag, it's famiiar, short, and it often makes sense - but less so when you're selling rather than buying things. P directives can represent costs, too. 

The docs prefer "transaction price", which is directionless and also hints at the similarity with and difference from P "market prices". Worth using that term instead ? --transaction-price is long, I'll admit, but the meaning is more obvious.

For "inverted", I'm still a bit confused even reading the test. A little help text or brief use case description could help to explain and motivate these I think.